composer.json

```diff
@@ -8,7 +8,9 @@
             "name": "Jeppe Zapp"
         }
     ],
-    "require": {},
+    "require": {
+      "ext-libxml": "*"
```

Comment:
I included this as IntelliJ told me so. Some PHP expert has to decide whether this is fine, or good.

Reply:
This look pretty fine to me after a quick research. I'd stick with that.